mlflow/sklearn/__init__.py

```diff
@@ -184,11 +197,14 @@ def save_model(
         code_paths: {{ code_paths }}
         mlflow_model: :py:mod:`mlflow.models.Model` this flavor is being added to.
         serialization_format: The format in which to serialize the model. This should be one of
-            the formats listed in
-            ``mlflow.sklearn.SUPPORTED_SERIALIZATION_FORMATS``. The Cloudpickle
+            the formats "skops", "cloudpickle" or "pickle".
+            The "skops" format guarantees safe deserialization.
+            The "cloudpickle"
             format, ``mlflow.sklearn.SERIALIZATION_FORMAT_CLOUDPICKLE``,
             provides better cross-system compatibility by identifying and
             packaging code dependencies with the serialized model.
+            The default format is "cloudpickle", but in future MLflow version, the default format
+            will be changed to "skops" for safe deserialization.
```

Comment:
changing default format to skops is a breaking change (skops need to set trusted types for custom model)
so I think we'd better change default in the version after next release.